### DISCUSSION TITLE
How to document splitters and splitter connections

### DISCUSSION THREAD

**Discussion by gregoryeid** [netbox-community/netbox#17507] [Help Wanted!]
I have one GPON OLT with 8 ports. PON 1 connects to a 1:2 splitter. Each output of the 1:2 splitter connects to a 1:32 splitter. What should I do to represent this in Netbox?

**Comment by Myke** [comment]
I wondered the same, created both interface-flavoured PLCs and patch panel-flavoured PLCs. 

But somewhere in the 3.x series, the ability to asset bridged interfaces and the ability for a trace to pass through it - works pretty well for me. (It does fall apart for the second input on a 2:4 however. But that's pretty minor compared to the power everything else gives.)

Here's a trace from an ONU to OLT:
![Screenshot 2024-09-20 at 16-34-49 NetBox](https://github.com/user-attachments/assets/a2591dda-b457-4570-bf60-fd8cc059aae9)

**Comment by SoerenBusse** [comment]
@Myke Ran into the same requirement and tried to replicate your setup. However I only managed to do a cable trace one way from output to input (as in your image). Were you able to find a solution to trace in both directions? If yes, could you maybe explain how you archived that?

**Reply by Myke** [reply]
You can't really; given from input to output could have multiple paths to follow.

**Comment by joaolucasmacedo** [comment]
> I have one GPON OLT with 8 ports. PON 1 connects to a 1:2 splitter. Each output of the 1:2 splitter connects to a 1:32 splitter. What should I do to represent this in Netbox?

You should use front and rear ports. In a 1:32 splitter yo would have a rear port with 32 positions and 32 corresponding front ports.
